static/manifests/calico/DaemonSet/calico-node.yaml

```diff
@@ -166,6 +166,9 @@ spec:
             # Enable or Disable VXLAN on the default IP pool.
             - name: CALICO_IPV4POOL_VXLAN
               value: "Never"
```

Comment:
actually our internal docs provides an example with using {{ .Overlay }} field for the CALICO_IPV4POOL_VXLAN instead of the hardcoded Never.

I wonder, was it "never"ed by some mistake in the past..



https://github.com/k0sproject/k0s/blob/main/docs/internal/upgrading-calico.md

https://docs.k0sproject.io/v1.27.4+k0s.0/configuration/#specnetworkcalico

Reply:
but that requires `mode` set to `vxlan` and not `ipip`?